package.json

```diff
@@ -4,7 +4,8 @@
   "main": "dist/lib.js",
   "license": "MIT",
   "scripts": {
-    "test": "",
+    "test": "jest --coverage",
```

Comment:
May want to switch this command to `jest` and make `test` do something like `npm jest && npm lint` (since lint runs tsc)

Reply:
If you want paralellism for the `test` command a quick tool to use is `concurrently`
https://github.com/kimmobrunfeldt/concurrently